source/jormungandr/tests/check_utils.py

```diff
@@ -495,6 +495,35 @@ def is_valid_journey(journey, tester, query):
         assert 'debug' not in journey
 
 
+def is_valid_geojson_coord(coord):
+    is_valid_lon(coord[0])
+    is_valid_lat(coord[1])
+
+
+def is_valid_geojson(geojson):
+    assert geojson['type'] == 'MultiPolygon'
+    assert geojson['coordinates']
+    for p in geojson['coordinates']:
+        outer = p[0]
+        for c in outer:
+            is_valid_geojson_coord(c)
+        for inner in p:
+            if inner != p[0]:
+                for c in inner:
+                    is_valid_geojson_coord(c)
+
+
+def is_valid_graphical_isochron(isochron, tester, query):
+
+    assert isochron['isochrons']
+    for g in isochron['isochrons']:
```

Comment:
to avoid doing this you can use:
`for g in get_non_null(isochron, 'isochrons')`, the assert will be done by the function